integrations/amazonKeys/cli/init.sh

```diff
@@ -1,99 +1,106 @@
 #!/bin/bash -e
-readonly IFS=$'\n\t'
```

Comment:
- why is this removed? .we need it for more predictable iteration of the arrrays

Reply:
We now have this in `common/_helpers.sh` ( https://github.com/Shippable/execTemplates/pull/16/files/d3d2c8c3206fa381c90c07050f78da0283453498#diff-569876d2c3cb06a576a7d4506fd4537fR3 ), since it is going to be sourced everywhere and we wanted this variable everywhere.